python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -482,10 +520,10 @@ def _get_threshold(nth_order):
             freeParameters = ((nth_order + 1) * (nth_order + 2)) // 2
             return freeParameters
 
-        if self.config.zerothOrderInterpNotEnoughStars:
-            if piffConfig['interp']['type'] in ['BasisPolynomial', 'Polynomial']:
-                threshold = _get_threshold(piffConfig['interp']['order'])
-                if len(stars) <= threshold:
+        if piffConfig['interp']['type'] in ['BasisPolynomial', 'Polynomial']:
+            threshold = _get_threshold(piffConfig['interp']['order'])
+            if len(stars) <= threshold:
```

Comment:
Quick fly-by comment since I noticed (and changed) this on DM-49673.  To my mind, this should be just `<` not `<=`.  I noticed when getting an error message that didn't seem to make sense along the lines:

```
Only 15 stars found, but 15 required. Using zeroth order interpolation.
```